lib/CMakeLists.txt

```diff
@@ -254,6 +254,7 @@ set(lib_HEADERS
 		filesystem/ISimpleResourceLoader.h
 		filesystem/MinizipExtensions.h
 		filesystem/ResourceID.h
+		filesystem/find_steam_game.h
```

Comment:
I included the whole header for convenience; open to changing how this file is included
For example in devilutionx they had a 3rd party CMakeLists.txt which downloaded the header from github https://github.com/diasurgical/devilutionX/commit/6091c3f3c2fbb68df4c463f5b92e57ce0850e62b